lib/galaxy/datatypes/util/gff_util.py

```diff
@@ -42,6 +42,8 @@ def __init__(self, reader, fields, chrom_col=0, feature_col=2, start_col=3, end_
         self.score = self.fields[self.score_col]
 
         # GFF attributes.
+        if 8 >= self.nfields:
+            raise MissingFieldError("No field for attribute column (8)")
```

Comment:
```suggestion
            raise MissingFieldError("No field for attribute column (9)")
```
It's the ninth column we're validating, right ?

Reply:
Also in this case I'm using the 0-indexed column number as in the code above and as used also in bx-python. Not sure it's useful to fix only in this method.

Reply:
I think if it's a user-facing exception it'd be clearer to use 1-based counting, but either way is fine.